### DISCUSSION TITLE
Spanish transcriptions contain text "This is the Spanish subtitles channel of The Church of Jesus Christ of Latter-day Saints"

### DISCUSSION THREAD

**Discussion by ivansabik** [openai/whisper#1699] [Q&A]
When transcribing multiple videos using Spanish language, I am getting this in multiple videos:

"Este es el canal de subtítulos en español de la Iglesia de Jesucristo de los Santos de los Últimos días"

This translates to something like: 

"This is the Spanish subtitles channel of the The Church of Jesus Christ of Latter-day Saints"

I am using a really simple code:
```python
whisper_model = whisper.load_model("medium")
transcription = whisper_model.transcribe(audio_file, language="es")
```

Examples:
- https://www.youtube.com/watch?v=0KTa6RgCI-I
- https://www.youtube.com/watch?v=RpR0LGUA3zg

Anybody has an idea of how we can solve this or where it could be coming from?

**Comment by glangford** [comment]
Could be the same underlying issue as
* https://github.com/openai/whisper/discussions/928

**Comment by ivansabik** [comment]
Definitely the same, thanks for pointing to that one
